apps/condo/domains/marketplace/components/Invoice/BaseInvoiceForm.tsx

```diff
@@ -502,7 +502,7 @@ const ServicesList = ({ organizationId, propertyId, form, currencySymbol, disabl
             const priceObj = get(pricesArray, '0')
             const price = get(priceObj, 'price')
             const isMin = get(priceObj, 'isMin')
-            const measure = get(priceObj, 'measure')
+            const measure = get(priceObj, 'measure', undefined)
```

Comment:
Just FYI
if `priceObj.measure` is `null` `get` will return `null`
`undefined` is default value for 3rd parameter, so nothing changed here
Did you meant:
```suggestion
            const measure = get(priceObj, 'measure') || undefined
```
?

Reply:
Thanks